### PR TITLE
run uninstall-profile on uninstalling

### LINKED ISSUE #4: Run an uninstall-profile on uninstalling
``Products.CMFQuickInstallerTool.InstalledProduct.InstalledProduct.uninstall`` should look for a and import a profile called ``uninstall`` if ``reinstall`` is ``False`` and ``getUninstallMethod`` does not find a method in ``Extensions/Install.py``. 

See https://github.com/plone/bobtemplates.plone/issues/27

### PR DESCRIPTION
**Pull request by pbauer**
This fixes #4. Do not merge yet since it is not tested (see #5).

### CODE REVIEW

**Comment by pbauer** [comment]
This is now ready to be merged.

**Comment by vangheem** [comment]
Thanks!

File: Products/CMFQuickInstallerTool/utils.py [hunk 1]
```diff
@@ -22,6 +22,8 @@
     'ZODBMountPoint', 'ZReST', 'ZSQLMethods',
 ])
 
+UNINSTALL_ID = 'uninstalled'
```
Comment (djowett) [inline Products/CMFQuickInstallerTool/utils.py]: @pbauer Any reason that UINSTALL_ID is declared here as well as in QuickInstallerTool.py?

Comment (pbauer) [inline Products/CMFQuickInstallerTool/utils.py]: Ups! That is totally obsolete.

**Comment by mauritsvanrees** [comment]
For the record, I have released 3.0.9 with this. Will probably end up in Plone 4.3.7 and 5.0b3 once they get released.

File: Products/CMFQuickInstallerTool/utils.py (hunk 1 above)
Comment (pbauer) [inline Products/CMFQuickInstallerTool/utils.py]: @djowett I removed it.

**Pull request merged**